manim/mobject/functions.py

```diff
@@ -38,7 +38,7 @@ def construct(self):
                         1.2 * np.cos(u),
                         1.2 * np.sin(u),
                         u * 0.05
-                    ]), color=RED, t_min=-3 * TAU, t_max=5 * TAU,
+                    ]), color=RED, t_range = np.array([-3*TAU, 5*TAU, 0.01])
```

Comment:
Why do you use `np.array` for one range and a normal `list` for the other?

Reply:
Oh no... I simply forgot the upper example when changing it first (searching for `t_min`) and then quickly fixed it, not remembering that `np.array` is used in the rest of the code.